### ISSUE TITLE
jqui_resizable cannot return the current size in time

### ISSUE THREAD

**Issue by Jonesmacke2** [Yang-Tang/shinyjqui#77]
jqui_resizable cannot return the current size in time. Usually the returned size is the last adjusted size.

**Comment by Yang-Tang** [comment]
Hi @Jonesmacke2 , with the commit 0b35989f4e17d5ca11a034af16fe393eef0a584b, `jqui_resizable` is now rerunning the correct size. Let me know if there are any problems.

**Comment by Jonesmacke2** [comment]
Thanks to the author for maintaining the code efficiently! There is another bug: when jqui_resizable was used in renderUI, jqui_resizable cannot get the correct plotOutput ID, but the uiOutput ID

**Comment by Yang-Tang** [comment]
Hi @Jonesmacke2 Thanks again for identifying the new issue. The ID problem is now fixed with ac8ad77d700106d1a17d0d99f83095993e3a1d64

**Comment by drag05** [comment]
My plot resizes only after browser page is toggled between "full page" and "normal page" (F11 key)

Should this be open as new issue or is this a browser issue?

I have a shiny app built in modules. 

The `module UI` is 
```
plotUI = function(id, label = 'plots') {
        ns = NS(id)
             ui = tagList(
             plotOutput(ns('spc')
                    , height = '450px'
                    , width = '880px'
      )
  )
}
```

The output  of this module in the `app UI` is 
```
mainPanel( withSpinner(plotUI('spc'), type = 7, color = '#5104e1') )
```

The `app server` includes
```
jqui_resizable(ui = ".shiny-plot-output", operation = 'enable')  ## the first line of code in app server

# plot SPC charts
plotServer('spc', data = fdt())  ## the module plotServer
```
I have tried with and without spinner and got the same result: able to drag the corner to any size but then I have to toggle the browser page to resize the plot on screen. Also, the spinner functionality disappears once I implement resizable plot. Please advise, thank you!
